salt/utils/virtualbox.py

```diff
@@ -281,7 +281,10 @@ def vb_get_network_addresses(machine_name=None, machine=None):
     # We can't trust virtualbox to give us up to date guest properties if the machine isn't running
     # For some reason it may give us outdated (cached?) values
     if machine.state == _virtualboxManager.constants.MachineState_Running:
-        total_slots = int(machine.getGuestPropertyValue('/VirtualBox/GuestInfo/Net/Count'))
+        try:
+            total_slots = int(machine.getGuestPropertyValue('/VirtualBox/GuestInfo/Net/Count'))
+        except ValueError:
+            total_slots = 0
```

Comment:
wouldn't it be easier to do something like.
```
if total_slots:
    total_slots = int(total_slots)
else:
    total_slots = 0
```
Instead of having to catch an exception?

Otherwise, this seems reasonable.

Reply:
I'm happy with either, as long as the guest property is only ever `''` or a digit